source/.vscode/extensions.json

```diff
@@ -1,5 +1,11 @@
 {
   // See http://go.microsoft.com/fwlink/?LinkId=827846
   // for the documentation about the extensions.json format
-  "recommendations": ["dbaeumer.vscode-eslint", "amodio.tsl-problem-matcher"]
-}
+  "recommendations": [
```

Comment:
Maybe do you considerer add the following recommendations

- mgmcdermott.vscode-language-babel
- vscode-icons-team.vscode-icons
- visioncan.vscode-jss-snippets